spec/airport_spec.rb

```diff
@@ -1,28 +1,34 @@
 require 'airport'
-
-## Note these are just some guidelines!
-## Feel free to write more tests!!
-
-# A plane currently in the airport can be requested to take off.
-#
-# No more planes can be added to the airport, if it's full.
-# It is up to you how many planes can land in the airport
-# and how that is implemented.
-#
-# If the airport is full then no planes can land
+require 'plane'
 
 describe Airport do
+  
+  subject { Airport.new }
+  subject { Plane.new }
+
+   it "has a default capacity" do
+    expect(airport.capacity).to eq 10
+  end
```

Comment:
Lint/BlockAlignment: `end` at 11, 2 is not aligned with `it "has a default capacity" do` at 9, 3